packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -420,6 +420,9 @@ export async function createMember(parent: api.IZoweDatasetTreeNode, datasetProv
                 await ZoweExplorerApiRegister.getMvsApi(profile).createDataSetMember(dsMemberName, {
                     responseTimeout: profile.profile?.responseTimeout,
                 });
+                if (replace === "replace") {
+                    deleteTempFile(dsMemberName, parent);
```

Comment:
We should probably `await` this function call, mostly so that the temp file is gone before `openDs` is called.

Reply:
Sorry for the confusion, the function didn't need to be async.
resolved in 3c3aee8f126d3f05d5bbc8c485209100af444306